.husky/pre-commit

```diff
@@ -1,9 +1,7 @@
 #!/usr/bin/env sh
-. "$(dirname -- "$0")/_/husky.sh"
-
-npm run format:fix
-npm run lint:fix
+# . "$(dirname -- "$0")/_/husky.sh"
 
+npx lint-staged
 git add .
 
 if [ -f "yarn.lock" ] || [ -f "package-lock.json" ]; then
```

Comment:
can we raise a warning? When user tries to commit `yarn.lock` or `package-lock.json`

Reply:
![image](https://github.com/user-attachments/assets/1c21c916-ad59-4e44-a4e5-bf341c26cbde)